UI_DSM/UI_DSM.Client/Program.cs

```diff
@@ -40,13 +41,12 @@ public static async Task Main(string[] args)
             builder.RootComponents.Add<HeadOutlet>("head::after");
 
             builder.Services.AddScoped(_ => new HttpClient
-                { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+                { BaseAddress = new Uri("https://localhost:7032/api/") });
```

Comment:
Are you sure? Hard coded instead of an Environment setting?

Reply:
I'll put it on the Environnement for the next PR